tests/numeric/test_PCATransformer.py

```diff
@@ -0,0 +1,416 @@
+            "d": [25.94, 70.22, 72.94, 64.55, 0.41, 13.62, 30.22, 4.6, 67.13, 10.38],
+            "e": [94.3, 4.18, 51.7, 16.63, 2.6, 16.57, 3.51, 30.79, 66.19, 25.44],
+            "pca0": [
+                -7.0285210087722,
+                -10.570772171093276,
+                0.7141476951788169,
+                -19.75551737702969,
+                30.46293987797488,
+                -37.272002248659426,
+                -37.718068808834694,
+                55.63624699948385,
+                -23.564287941836838,
+                49.09583498358856,
+            ],
+            "pca1": [
+                -14.71905708522354,
+                0.6588448890236054,
+                -6.5048093686104504,
+                8.411936495027229,
+                30.755961905144936,
+                -0.8912674725933969,
+                -2.647964525208771,
+                -9.600190936709119,
+                2.660636497589114,
+                -8.124090398439632,
+            ],
+        }
+    )
+    return svd_sovler_output
+
+
+class TestTransform(object):
+    """Tests for PCATransformer.transform()."""
+
+    def test_arguments(self):
+        """Test that transform has expected arguments."""
+
+        ta.functions.test_function_arguments(
+            func=PCATransformer.transform,
+            expected_arguments=["self", "X"],
+            expected_default_values=None,
+        )
+
+    def test_super_transform_called(self, mocker):
+        """Test that BaseTransformer.transform called."""
+
+        df = created_numeric_df_1()
+
+        x = PCATransformer(columns=["a", "b"])
+
+        x.fit(df)
+
+        expected_call_args = {0: {"args": (created_numeric_df_1(),), "kwargs": {}}}
+
+        with ta.functions.assert_function_call(
+            mocker,
+            tubular.base.BaseTransformer,
+            "transform",
+            expected_call_args,
+            return_value=created_numeric_df_1(),
+        ):
+
+            x.transform(df)
+
+    def test_check_numeric_columns_call(self, mocker):
+        """Test the call to PCATransformer.check_numeric_columns."""
+
+        df = created_numeric_df_1()
+
+        x = PCATransformer(columns=["a", "b"], copy=True)
+
+        x.fit(df)
+
+        expected_call_args = {0: {"args": (created_numeric_df_1(),), "kwargs": {}}}
+
+        with ta.functions.assert_function_call(
+            mocker,
+            tubular.base.BaseTransformer,
+            "transform",
+            expected_call_args,
+            return_value=created_numeric_df_1(),
+        ):
+
+            x.transform(df)
+
+    @pytest.mark.parametrize(
+        "svd_solver, svd_solver_output_str",
+        [("full", "full"), ("arpack", "arpack"), ("randomized", "randomized")],
+    )
+    def test_output_from_scaler_transform_set_to_columns(
+        self, mocker, svd_solver, svd_solver_output_str
+    ):
+        """Test that the call to the pca.transform method."""
```

Comment:
"""Test that the call to the pca.transform method returns expected outputs."""

Reply:
Commit : [udate name and description of test_output](https://github.com/lvgig/tubular/pull/57/commits/425c33dd9998fa75c5c2aec4ad11ed9cdcdc2a64)